.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: cpp
-sudo: false
+dist: trusty
```

Comment:
Is this required now?

Reply:
No, I put it to be more specific, and to prevent changes in CI when the default changes